core/tests/protractor_desktop/navigation.js

```diff
@@ -172,4 +172,54 @@ describe('Static Pages Tour', function() {
     expect(element(
       by.css('.protractor-test-splash-page')).isPresent()).toBe(true);
   });
+
+  it('visits the About page', function() {
+    browser.get('/about');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-about-page')).isPresent()).toBe(true);
+  });
+
+  it('visits the Contact page', function() {
+    browser.get('/contact');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-contact-page')).isPresent()).toBe(true);
+  });
+
+  it('visits the Donate page', function() {
+    browser.get('/donate');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-donate-page')).isPresent()).toBe(true);
+  });
+
+  it('visits the Privacy page', function() {
+    browser.get('/privacy');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-privacy-page')).isPresent()).toBe(true);
+  });
+
+  it('visits the Terms page', function() {
+    browser.get('/terms');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-terms-page')).isPresent()).toBe(true);
+  });
+
+  it('visits the Thanks page', function() {
+    browser.get('/thanks');
+    waitFor.pageToFullyLoad();
+    expect(element(
+      by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
+  });
+
+  it('redirects to the test server', function() {
```

Comment:
> @Jamesjay4199 any reason why this test was deleted? 

@lilithxxx, the test was moved here.
It wasn't exactly deleted.

Reply:
But this made the backend coverage decrease. You can have two tests I think

Reply:
> But this made the backend coverage decrease.

Codeclimate didn't show this.

Reply:
The page that was tested there, can not be tested via a backend test because it is now served via app.yaml no longer python

Reply:
> Codeclimate didn't show this.

Yes I am looking into it

> The page that was tested there, can not be tested via a backend test because it is now served via app.yaml no longer python

Yes I understand that but all I am saying is that you can have 2 tests right? Will this existing python test fail now?

Reply:
> Will this existing python test fail now?

Yes. The python test will fail since the page that was tested isn't loaded through python.